tests/integration/raw_studies_blueprint/test_aggregate_raw_data.py

```diff
@@ -9,188 +9,243 @@
 from antarest.study.storage.rawstudy.model.filesystem.matrix.matrix import MatrixFrequency
 from tests.integration.raw_studies_blueprint.assets import ASSETS_DIR
 
+AREAS_REQUESTS = [
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": AreasQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "",
+            "areas_ids": "",
+            "columns_names": "",
+        },
+        "test-01.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": AreasQueryFile.DETAILS,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "1",
+            "areas_ids": "de,fr,it",
+            "columns_names": "",
+        },
+        "test-02.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": AreasQueryFile.VALUES,
+            "frequency": MatrixFrequency.WEEKLY,
+            "mc_years": "1,2",
+            "areas_ids": "",
+            "columns_names": "OP. COST,MRG. PRICE",
+        },
+        "test-03.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": AreasQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "2",
+            "areas_ids": "es,fr,de",
+            "columns_names": "",
+        },
+        "test-04.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": AreasQueryFile.VALUES,
+            "frequency": MatrixFrequency.ANNUAL,
+            "mc_years": "",
+            "areas_ids": "",
+            "columns_names": "",
+        },
+        "test-05.result.tsv",
+    ),
+]
+
+LINKS_REQUESTS = [
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": LinksQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "",
+            "columns_names": "",
+        },
+        "test-01.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": LinksQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "1",
+            "columns_names": "",
+        },
+        "test-02.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": LinksQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "1,2",
+            "columns_names": "UCAP LIN.,FLOW QUAD.",
+        },
+        "test-03.result.tsv",
+    ),
+    (
+        {
+            "output_id": "20201014-1425eco-goodbye",
+            "query_file": LinksQueryFile.VALUES,
+            "frequency": MatrixFrequency.HOURLY,
+            "mc_years": "1",
+            "links_ids": "de - fr",
```

Comment:
We need to choose if we use the hyphen or the percent sign as a delimiter.

Reply:
I think we can keep this as it is but I don't mind changing it.